.github/ISSUE_TEMPLATE/3.rfc.yml

```
@@ -0,0 +1,47 @@
+name: RFC 💬
+description: Request for comments for your proposal.
+title: "[RFC] "
+labels: ['RFC']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the RFC in the title above ⬆️.
```

Punctuation before emoji.

```suggestion
        Please provide a searchable summary of the RFC in the title above. ⬆️
```